app/routes.py

```diff
@@ -37,6 +37,17 @@ def total_recipes():
 
 #---------- APP ROUTES ----------#
 
+#----- 404 -----#

```

Comment:
Use Python docstrings to give information on your functions https://www.python.org/dev/peps/pep-0257/

Reply:
Yessir! I plan on adding the **tests** and **docstrings** after I refactor all of the code.
First build and break it, then refactor and clean it up!
Thanks!